Sources/ArgumentParser/Documentation.docc/Extensions/AsyncParsableCommand.md

```diff
@@ -29,14 +29,22 @@ struct CountLines: AsyncParsableCommand {
 
 ### Usage in Swift 5.5
```

Comment:
@natecook1000 what do you think about just dropping this content from the docs? Current ArgumentParser doesn't support 5.5 and users of a version that does can reference older documentation instead.

Reply:
@rauhul @natecook1000 What can I do to move this forward?

Thanks.

Reply:
@rauhul @natecook1000 I'll be happy to modify this PR however you want to get it resolved.

If you're not sure if you want to delete the 5.5 instructions, it might make sense to accept this PR as is, since it improves the 5.5 instructions. You can then decide whether to keep or delete the 5.5 instructions at your leisure, but at least everything will be updated in the meantime.

Reply:
Apologies for the delay on this, @rgoldberg! I think Rauhul is right that we don't need this in the mainline docs anymore, but I don't want it to completely disappear. I can't decide between making a separate article and sticking in the changelog, so why don't we get this merged and I can move it later on.

Thanks!

Reply:
Thanks for merging this!